src/app/public/modules/list-view-grid/list-view-grid.component.ts

```diff
@@ -157,7 +157,7 @@ export class SkyListViewGridComponent
     }
 
     // Setup Observables for template
-    this.columns = this.gridState.map(s => s.columns.items).distinctUntilChanged();
+    this.columns = this.gridState.map(s => s.columns.items).distinctUntilChanged(this.arraysEqual);
```

Comment:
Should this also have a `.takeUntil(this.ngUnsubscribe)`?

Reply:
Done